live-examples/css-examples/transforms/translate.html

```diff
@@ -0,0 +1,42 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="translate">
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">translate: 50%;</code></pre>
```

Comment:
I think it might be useful to show `none` also. And to demonstrate how it works with 1-3 values. How about:

* `translate: none;`
* `translate: 50%;` one value
* `translate: 50% 2rem;` two values
* `translate: 50% 2rem -20px;` three values
* ... (optional if we want to limit the height of the example)

What do you think?

Reply:
That sounds good. I have updated example to match your suggestions.

Reply:
That sounds good. I have updated the example to match your suggestions.